src/Plugins.js

```diff
@@ -107,11 +113,9 @@ async function login({page, options} = {}) {
 }
 
 async function typeUsername({page, options} = {}) {
-  let buttonSelector = options.headless ? '#next' : '#identifierNext'
-
   await page.waitForSelector('input[type="email"]')
   await page.type('input[type="email"]', options.username)
-  await page.click(buttonSelector)
+  await page.click('#identifierNext')
```

Comment:
why hard-coding it to that though?
it makes a decision of which to use based on headless property in line 110 on the left. is that wrong?

Reply:
Since the page headers get set now, it no longer needed to switch between the two properties. Or is something breaking that I'm not realizing?